app/scripts/use-snow.js

```diff
@@ -12,16 +12,45 @@ Changing this code must be done cautiously to avoid breaking the app!
   const isWorker = !self.document;
   const msg =
     'Snow detected a new realm creation attempt in MetaMask. Performing scuttling on new realm.';
+
+  // eslint-disable-next-line no-undef
+  const chromeExtensionId = chrome && chrome.runtime && chrome.runtime.id;
+  const tamedFetch = (path) => {
+    const regex = new RegExp(
+      `^chrome-extension://${chromeExtensionId}/.*?/images/.*`,
+      'u',
+    );
+    if (regex.test(path)) {
+      return fetch(path);
+    }
+    throw new Error(
+      'Unscuttled fetch can only be used on images within the extension',
+    );
+  };
+
   // eslint-disable-next-line no-undef
   Object.defineProperty(self, 'SCUTTLER', {
     value: (realm, scuttle) => {
+      const scuttleWithRestrictedException = (_ref, restrictedExceptions) => {
+        const ref = _ref;
+        for (const key in restrictedExceptions) {
+          if (Object.hasOwn(restrictedExceptions, key)) {
+            ref[key] = restrictedExceptions[key];
```

Comment:
We must use defineProperty and set configurable and writable to false. This will instruct the scuttling process to skip `fetch`, which would allow you to remove the exception from `development/build/index.js`

Reply:
done in 1dd34f9af5